nucypher_ops/playbooks/include/run_porter.yml

```diff
@@ -12,7 +12,7 @@
         restart: yes
         image: "{{ docker_image }}"
         restart_policy: "unless-stopped"
-        command: 'nucypher porter run --eth-provider {{ eth_provider }} {{CLI_RUNTIME_OPTIONS}}'
+        command: 'nucypher-porter porter run --eth-provider {{ eth_provider }} {{CLI_RUNTIME_OPTIONS}}'
```

Comment:
😂 Probably better to direct this fix at porter!

Reply:
https://github.com/nucypher/nucypher-porter/pull/13

Reply:
Thanks!  I'll do some live testing of this and make sure it all works before I push a release.

Reply:
Just a heads up that once PR #13 is merged this change will be obsoleted.

Reply:
@theref FYI - the aforementioned porter PR has now been merged.